opentracing_shim/src/test/java/io/opentelemetry/opentracingshim/testbed/TestUtils.java

```diff
@@ -179,8 +179,12 @@ public static void assertSameTrace(List<SpanData> spans) {
     for (int i = 0; i < spans.size() - 1; i++) {
       // TODO - Include nanos in this comparison.
       assertTrue(spans.get(spans.size() - 1).getEndEpochNanos() >= spans.get(i).getEndEpochNanos());
-      assertEquals(spans.get(spans.size() - 1).getTraceId(), spans.get(i).getTraceId());
-      assertEquals(spans.get(spans.size() - 1).getSpanId(), spans.get(i).getParentSpanId());
+      assertEquals(
+          spans.get(spans.size() - 1).getTraceId().toString(),
```

Comment:
Don't need `toString()` (presumably it's a better assertion without it too). Next few files have it